### PR TITLE
Checking "commit_12_31_2022_6:34:PM" from "Hp7256coder" user

### PR DESCRIPTION
**Pull request by phuongtrieu97coder**
Thank you for adding some output message into the "return" syntax on line 3 of Test1.js file in this repository
image

### CODE REVIEW

File: Test1.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 function test1 (){
 
-    return 
+    return "Hello world";
```
Comment (Hp7256coder) [inline Test1.js:3]: I edited this syntaxes for testing purpose

**Review by Hp7256coder — APPROVED** [review]
I've already checked your pull request. Everything is fine.

File: Test1.js (hunk 1 above)
Comment (phuongtrieu97coder) [reply Test1.js:3]: Thanks!!!

**Review by Hp7256coder — COMMENTED** [review]
I have viewed this pull request

**Review by phuongtrieu97coder — COMMENTED** [review]
Approved.

**Review by Hp7256coder — CHANGES_REQUESTED** [review]
suggesting changes

**Review by phuongtrieu97coder — COMMENTED** [review]
Succesfully

**Review by Hp7256coder — APPROVED** [review]
Approve

**Review by Hp7256coder — APPROVED** [review]

**Review by Hp7256coder — COMMENTED** [review]
Good

**Comment by Hp7256coder** [comment]
good

**Review by Hp7256coder — APPROVED** [review]
Good

**Pull request merged**